documentation/docs/post_installation/managing_a_node.md

```diff
@@ -1,3 +1,14 @@
+---
+keywords:
```

Comment:
I don't know whether or not it makes an impact but some keywords are capitalized in other sections while some are not. Probably not a big deal, just something I noticed!

Reply:
I don't think it makes a difference